source/best-practices/backup.rst

```diff
@@ -1,61 +1,133 @@
 备份与还原
 ==========
 
+:本节贡献者: |田冬冬|\（作者）、
+             |姚家园|\（作者）
+:最近更新日期: 2021-01-30
+:预计阅读时间: 20 分钟
+
+----
+
 **备份！备份！备份！**
 
 尽管不经常发生，但电脑随时可能会坏掉或被盗，电脑硬盘也随时可能会出问题，
-也可能不小心使用一个命令把几个月的工作误删了。所以，备份非常重要，不进行必要的备份
-是十分危险的。
+也可能不小心使用一个命令（如 ``rm *``）把几个月的工作误删了。所以，备份非常重要，不进行必要的备份
+是十分危险的。笔者和一些同行多次体验过数据误删或硬盘损坏又未备份带来的痛苦。
 
-备份的目的
+备份的策略
 ----------
 
+备份的策略简单概括有三种：
+
 - 仅备份重要文件
 - 仅备份家目录
 - 全盘备份
 
+不同备份策略的主要区别是所需的硬盘空间和备份时间。例如，若仅备份一些重要文件，
+则需要的硬盘空间较短，备份所需要的时间也较小。若要全盘备份，则需要较大的硬盘空间和
+较长的备份时间。仅备份家目录介于二者之间。
+
+建议用户根据个人需求和存储空间来决定具体的备份策略。如果某些文件非常重要，
+丢失后恢复难度极大或恢复时间极长，强烈推荐备份一下，如毕业论文、无法再次获取的数据、
+经过长时间处理得到的资料。如果拥有很大的硬盘存储空间，可以进一步选择备份家目录，
+甚至全盘备份。
+
 备份的介质
 ----------
 
-- 硬盘备份
-- 网盘备份
+我们需要一定的存储空间来备份电脑里的文件，常用的备份介质有\ **硬盘**\ 和\ **网盘**\ 。
+
+最常见的备份介质是移动硬盘，其特点是携带方便、价格便宜。近几年，移动硬盘发展迅速，市场上
+常见的存储空间大小有 4TB 和 5TB，基本可以满足日常科研的备份需求。如果不够那就买两块吧。
+一些研究组可能也会购买存储空间更大的磁盘阵列，一般由许多块几 TB 大小的硬盘组成，价格较贵。
+将磁盘阵列挂载到本地电脑后，使用起来就像一块超大的移动硬盘一样，非常方便。
+
+网盘本质上也是硬盘，只不过需要通过网络连接才可以使用，大多数网盘都有客户端。本地客户端
+里的文件跟云端实时同步，用户基本不要做额外操作。因此，网盘的主要用途便是备份文件以及
+在不同设备之间同步文件。例如，可以把一份文件传到网盘上，在不同操作系统的客户端下都可以操作文件，
+且文件一直保持同步。
 
-我们主要讨论硬盘备份。
+目前比较流行的网盘有：
+
+- `坚果云 <https://www.jianguoyun.com>`__\ ：国内最好的全平台网盘。不限空间，
+  但限制流量，每月上传流量 1 GB，下载流量 3 GB。有需要的可以考虑购买高级版。
+- `百度网盘 <https://pan.baidu.com>`__\ ：全平台网盘，免费空间可以有 2 TB。
+- `Dropbox <https://www.dropbox.com/>`__\ ：国外最好用的全平台网盘，熟练掌握
+  科学上网技巧的人可以使用。免费用户只有 2 GB 容量，最大可扩容到 18 GB 以上。
+  付费用户容量为 1 TB。
+- `MEGA <https://mega.io/>`__\ ：全平台网盘，免费空间 50 GB。
+- `Google Drive <https://www.google.com/drive/>`__\ ：Google 云端硬盘。免费容量 15 GB，由 Google 云端网盘、
+  `Google 相册 <https://www.google.com/photos/>`__ \ 、Gmail 共用。
+- `OneDrive <https://www.microsoft.com/en-us/microsoft-365/onedrive/online-cloud-storage>`__\ ：
+  微软的云存储服务，初始免费容量 5 GB。Word、Excel、PowerPoint 以及 Onenote 等微软软件的文件
+  都可以直接保存到 OneDrive 中。
```

Comment:
Better to show a comparison using a table.

The information we may be interested:

1. Free quota and price for more quota
2. Cross-platform? Linux? macOS? Windows?
3. Easy to use in China?

Reply:
done in cde691d.